samples/c_cxx/model-explorer/batch-model-explorer.cpp

```diff
@@ -60,7 +60,9 @@ int main(int argc, char** argv) {
     cout << "Usage: ./onnx-api-example <onnx_model.onnx>" << endl;
     return -1;
   }
-  std::string model_file = argv[1];
+  std::string str = argv[1];
+  std::wstring wide_string = std::wstring(str.begin(), str.end());
+  std::basic_string<ORTCHAR_T> model_file = std::basic_string<ORTCHAR_T>(wide_string);
```

Comment:
need #ifdef condition, check out the ORTCHAR_T definition here:
[https://github.com/microsoft/onnxruntime/blob/743a93faf3dde7d7eef9180c360de05a94174154/include/onnxruntime/core/session/onnxruntime_c_api.h#L44-L65](https://github.com/microsoft/onnxruntime/blob/743a93faf3dde7d7eef9180c360de05a94174154/include/onnxruntime/core/session/onnxruntime_c_api.h#L44-L65)

Reply:
Aha. I see. 

Unfortunately I am working with onnxruntime on a Windows project and am not in a position to test on other OSs :\. I will update the PR with the appropriate ifdefs, but I can't promise they will work on Unix based OSs...

Reply:
#ifdef _WIN32
  std::string str = argv[1];
  std::wstring wide_string = std::wstring(str.begin(), str.end());
  std::basic_string<ORTCHAR_T> model_file = std::basic_string<ORTCHAR_T>(wide_string);
#else
  std::string model_file = argv[1];
#endif

---
In reply to: [590853818](https://github.com/microsoft/onnxruntime/pull/6950#discussion_r590853818) [](ancestors = 590853818)

Reply:
Ha! Yeah, that is better. I will push it.

Reply:
Although, it doesn't exemplify the meaning of ORTCHAR_T.... I guess it doesn't really matter though. That isn't the main point of the sample!